[WIP] Update onboarding docs (#5)
tsg-iitkgp/OnBoarding: TODOs.md

# TODOs

> In this doc, we track decisions that need to be made, or important information that needs to be passed on from the past technology coordinator. Add all TODOs with dates, and mark when they're finished or further action has been taken

Reviewer: ```suggestion
In this doc, we track decisions that need to be made or important information that needs to be passed on from the past technology coordinator. Add all TODOs with dates, and mark when they're finished or further action has been taken
```

## 17th July 2021

> 1. **Restart the Assimilation Program event: ** If not for the AP, we could conduct an event at the beginning of the academic session. This is to be done in coordination with the VP, KOSS, MetaKGP and the G.Sec Students' Welfare

Reviewer: ```suggestion
1. **Restart the Assimilation Program event: ** If not for the AP, we could conduct an event at the beginning of the academic session. This is to be done in coordination with the VP, KOSS, MetaKGP and the G.Sec Students' Welfare.
```

> 2. **Find a better solution to storing data: ** Right now, we update the JSON files in the repository using sheets given. For this, we usually write a script to bring the data in the right format, and it involves a lot of effort, customized to each sheet. We used to use a Gymkhana backserver to serve data from sheets, which we can revive and add all formatted data to the sheets directly. There could be other solutions as well, which can be discussed and tried. As the amount of data keeps increasing, it'll be very painful to manage all of it, and add it to the website

Reviewer: ```suggestion
2. **Find a better solution to storing data: ** Right now, we update the JSON files in the repository using the sheets given. For this, we usually write a script to bring the data in the right format, and it involves a lot of effort, customized to each sheet. We used to use a Gymkhana back server to serve data from sheets, which we can revive and add all formatted data to the sheets directly. There could be other solutions as well, which can be discussed and tried. As the amount of data keeps increasing, it'll be very painful to manage all of it and add it to the website.
```

> 3. **Standardize data formats for various events: ** Right now, we use Google Sheets for data, and all sheets use a different format to store data related to GC, InterIIT, and previous office bearers. Some of this is already done, using the formatted data sheet, but we need to document this format so that the G.Secs/Secretaries can update data according to this format

Reviewer: ```suggestion
3. **Standardize data formats for various events: ** Right now, we use Google Sheets for data, and all sheets use a different format to store data related to GC, InterIIT, and previous office bearers. Some of this is already done, using the formatted datasheet, but we need to document this format so that the G.Secs/Secretaries can update data according to this format.
```
